ynput/OpenPype
# Max: Validate loaded plugins tweaks

## Changelog Description
In the current development of 3dsMax, users need to use separate validators to validate if certain plugins being loaded before the extraction. For example, usd extractor in model family, prt/tycache extractor in pointcloud/tycache family.
But with the PR where implements optional validate loaded plugin, users just need to put what kind of plugins they want to validate in the settings. They no longer need to go through all the separate plugin validators when publishing, and only one validator would do all the check on the loaded plugins before extraction.

## Additional info
n/a

## Testing notes:
1. Set the plugin you want to check
![image](https://github.com/ynput/OpenPype/assets/64118225/54c3ac64-f4f6-45ae-ad5a-00e02e276909)
![image](https://github.com/ynput/OpenPype/assets/64118225/370b7c2e-b699-4b89-b708-d70d4bbafc7a)

2. Create any instance
3. Publish
4. If it doesn't pass the validation, and it is just not being loaded, you can use the repair action
5. If it doesn't pass the validation and it doesn't exist in 3dsmax Plugins list, you need to add the plugins inside the plugin folder  in your 3dsmax folder first.
